static/app/components/globalDrawer/components.tsx

```diff
@@ -63,9 +65,12 @@ export function DrawerPanel({
       drawerKey,
       drawerWidth,
     });
+  const theme = useTheme();
+  const isSmallScreen = useMedia(`(max-width: ${theme.breakpoints.small})`);
```

Comment:
Instead of returning this here, how about returning a `enabled` from the `useDrawerResizing`